editor/components/skip-to-selected-block/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+import { getBlockWrapperDOMNode } from '../../utils/dom';
+
+const SkipToSelectedBlock = ( { selectedBlock } ) => {
+	if ( ! selectedBlock ) {
+		return null;
+	}
+
+	const { uid } = selectedBlock;
+
+	const onClick = () => {
+		const selectedBlockElement = getBlockWrapperDOMNode( uid );
+		selectedBlockElement.focus();
+	};
+
+	return (
+		selectedBlock && uid &&
```

Comment:
`selectedBlock` will never be falsey here. You already have a condition above for `! selectedBlock`.

Reply:
`selectedBlock` is falsey initially and switching (even with mouse) the sidebar to the block inspector renders the skip link (the render method must return "something")